data/remnant/remnant.txt

```diff
@@ -790,10 +790,16 @@ shipyard "Remnant"
 	"Pelican"
 	"Petrel"
 	"Tern"
+	"Korath Dredger (Ground Assault)"
```

Comment:
What's the reasoning behind making this available from the get-go? It feels out of place, especially with the price tag more than doubling anything else available.

Reply:
That is completely a mistake. I just added it in to check that the crew attribute worked, and accidentally committed it. I'll delete it right away.

Reply:
Done. Thanks for spotting that!